### PR TITLE
Add manual quarantine tests

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**

### CODE REVIEW

**Comment by ShivamPokhriyal** [comment]
Looks like it's failing because of the `changeWifi` issue in browserstack. I'll keep this PR as open, till I find a workaround for that issue.

**Comment by shubham1g5** [comment]
@ShivamPokhriyal wondering if we can add `@supress` to it so that it gets skipped on BS and then implement another annotation say "@WifiDependent" and have a mechanism to run all these wifi dependent tests locally before every release.

**Comment by ShivamPokhriyal** [comment]
> @ShivamPokhriyal wondering if we can add `@supress` to it so that it gets skipped on BS and then implement another annotation say "@WifiDependent" and have a mechanism to run all these wifi dependent tests locally before every release.

Sounds really great. Will follow up on it in another PR.

File: app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        InstrumentationUtility.gotoHome()
+        InstrumentationUtility.selectOptionItem(withText("Advanced"))
+        onView(withText("Enable Manual Form Quarantine"))
+                .perform(click())
+        InstrumentationUtility.gotoHome()
+    }
+
+    @Test
+    fun test_A_Quarantine() {
+        InstrumentationUtility.openModule(12)
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt:54]: nit: if we are using position as input to `openModule`, we should put a comment here specifiying the text of module we want to open for better readability or move the value `12` in a constant whose name indicates what module this position is for.

File: app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        )).perform(longClick())
+        // Unsent forms should not be delete-able
+        withText("Delete Record").doesNotExist()
+        onView(withText("Scan Record Integrity"))
+                .perform(click())
+        onView(withText("QUARANTINE FORM"))
+                .perform(click())
+
+        // After quarantining one form we'll have to re-enable quarantine setting.
+        enableFormQuarantine()
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt:77]: we should add an additional step that just checks that you can't quarantine without this call.

File: app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        const val APP_NAME = "Basic Tests"
+    }
+
+    @Before
+    fun setup() {
+        installApp(APP_NAME, CCZ_NAME)
+        InstrumentationUtility.login("test", "1234")
+        // Enable quarantine
+        enableFormQuarantine()
+        InstrumentationUtility.changeWifi(false)
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/ManualQuarantineTest.kt:41]: Need to suppress below Android 10

**Review by shubham1g5 — CHANGES_REQUESTED** [review]

**Comment by ShivamPokhriyal** [comment]
@damagatchi retest this please

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462111/)

**Comment by ShivamPokhriyal** [comment]
@shubham1g5 Another review please.

**Review by shubham1g5 — APPROVED** [review]
Looks great.

**Pull request merged**